### PR TITLE
Add GPS:Info messageType

### PR DESCRIPTION
**Pull request by Frank-Wiebeler**

### CODE REVIEW

File: docs/tmt/gps.adoc [hunk 1]
```diff
@@ -0,0 +1,104 @@
...
+|9-13 |RESERVED |Values reserved
+|14 |D_ERROR | Error receiving GPS position from the GPS receiver
+|15 |D_NOT_AVAILABLE |No GPS Receiver available
+|================================================================================================
+
+The FieldStatus shall indicate, if a machine is on a field, on the road or offroad
+[cols="1,2,5",options="header",]
+|================================================================================================
+|# |Name |Description
+|0 |FS_UNKNOWN | The position status is not known
```
Comment (saschadoemer) [inline docs/tmt/gps.adoc:77]:
```suggestion
|0 |FS_UNKNOWN | The position status is not known, should be the default if there is no other status
```

File: docs/tmt/gps.adoc [hunk 2]
```diff
@@ -0,0 +1,104 @@
...
+
+
+*REMARK*
+====
+FS_UNKNOWN should be the default value if there is no other status known for sure.
+====
+
+=== Format
+
+The gps:info has **NOT** to be formatted to a Base64-encoding! It must be sent binary with a Technical message type gps:info set.
```
Comment (saschadoemer) [inline docs/tmt/gps.adoc:91]:
```suggestion
The `gps:info` has **NOT** to be formatted to a Base64 encoding! It must be sent binary with a technical message type `gps:info` set.
```

File: docs/tmt/gps.adoc (hunk 1 above)
[the remark this answers is not in the document]
Comment (Frank-Wiebeler) [reply docs/tmt/gps.adoc]: Good questions: It's about the position of a Communication Unit, but that is normally somehow attached to a device (not necessarily an agricultural machine but for example a car or truck). Device is the general term used in ISO11783 as well as EFDI. "The communication unit is on a field" sounds weird, doesn`t it?

[the remark this answers is not in the document]
Comment (saschadoemer) [reply docs/tmt/gps.adoc]: Yes, so we should use the term device for all of these points.

[the remark this answers is not in the document]
Comment (Frank-Wiebeler) [reply docs/tmt/gps.adoc]: Fixed

**Pull request merged**